tests/test_adhoc_tools.py

```diff
@@ -48,15 +44,13 @@ def test_get_default_interactive_config():
 
 
 def test_get_default_interactive_config_reads_from_tty():
-    with contextlib.nested(
-        mock.patch('paasta_tools.adhoc_tools.prompt_pick_one', autospec=True),
-        mock.patch('paasta_tools.adhoc_tools.load_adhoc_job_config', autospec=True),
-        mock.patch('paasta_tools.adhoc_tools.load_v2_deployments_json', autospec=True),
-    ) as (
-        mock_prompt_pick_one,
-        mock_load_adhoc_job_config,
-        mock_load_deployments_json,
-    ):
+    with mock.patch(
+        'paasta_tools.adhoc_tools.prompt_pick_one', autospec=True,
+    ) as mock_prompt_pick_one, mock.patch(
```

Comment:
Seems like this parses, but flake8 is unhappy:

        with \
            mock.patch('paasta_tools.check_marathon_services_replication.send_event', autospec=True) \
                as mock_send_event, \
            mock.patch('paasta_tools.marathon_tools.read_registration_for_service_instance', autospec=True,
                       return_value=compose_job_id(service, instance)), \
            mock.patch('paasta_tools.check_marathon_services_replication.load_smartstack_info_for_service', autospec=True) \
                as mock_load_smartstack_info_for_service, \
            mock.patch('paasta_tools.marathon_tools.load_marathon_service_config', autospec=True) \
                as mock_load_marathon_service_config\
        :